tests/monitoring/monitoring.go

```diff
@@ -93,10 +96,10 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			By("Migrating the VMI 13 times")
 			for i := 0; i < 13; i++ {
 				migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
-				migration = tests.RunMigrationAndExpectCompletion(virtClient, migration, tests.MigrationWaitTime)
+				migration = libmigration.RunMigrationAndExpectToComplete(virtClient, migration, libmigration.MigrationWaitTime)
```

Comment:
nit: avoid repeating default migration timeout here and elsewhere

```suggestion
				migration = libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, migration)
```